starter-code/src/App.js

```diff
@@ -1,18 +1,106 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import FoodBox from './components/FoodBox';
+import foods from './foods.json';
+import AddFood from './components/AddFood'
+import Search from './components/Search'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      foods, 
+      showForm: false,
+      todayFood: [],
+    }
+    this.addFood = this.addFood.bind(this)
+    this.toggleForm = this.toggleForm.bind(this)
+    this.filterFood = this.filterFood.bind(this)
+    this.addTodaysFood = this.addTodaysFood.bind(this)
+    this.deleteFood = this.deleteFood.bind(this)
+  }
+
+  addFood(food) {
+    const newFoods = [...this.state.foods]
+    newFoods.push(food)
+
+    this.setState({
+      foods: newFoods
+    })
+  }
+
+  toggleForm(){
+    this.setState({
+      showForm: !this.state.showForm
+    })
+  }
+
+  filterFood(searchLetters) {
+    const filteredFood = foods.filter(food=> {
+      return food.name.includes(searchLetters)
```

Comment:
```suggestion
      return food.name.toLowerCase().includes(searchLetters.toLowerCase())
```

normalizando a palavra buscada e os nomes dos elementos, vocês conseguem fazer com que a busca seja insensitive case.